server_src/routes/printers.js

```diff
@@ -102,7 +102,7 @@ router.post("/updateSettings", ensureAuthenticated, async (req, res) => {
   const settings = req.body;
   logger.info("Update printers request: ", settings);
   const updateSettings = await Runner.updateSettings(settings);
-
+  res.sendStatus(200);
```

Comment:
Why should we set a status if default is 200? Im a lazy .NET dev, all my status codes are automagically set XD

Reply:
I think this can be removed, I was chasing the settings issue and thought it was here. Will test and remove. Thought it was sending a 204 for some reason.

Reply:
Nothing wrong with a 204: just means the body is empty.

Reply:
I want a body, I specifically look for 204's in the new OctoFarm client.